slides.md

```diff
@@ -1053,10 +1053,87 @@ export default function profile(fastify) {
 
  </div>
 
+---
+
 # A10: Server Side Request Forgery
 
 <div class="dense">
 
+- SSRF flaws occur whenever a web application is fetching a remote resource without validating the user-supplied URL
+- It allows an attacker to coerce the application to send a crafted request to an unexpected destination, even when protected by a firewall, VPN, or another type of network access control list
+
+</div>
+
+---
+
+# A10: Server Side Request Forgery: How to prevent
+
+<div class="dense">
+
+- Sanitize and validate all client-supplied input data
+- Enforce the URL schema, port, and destination with a positive allow list
+- Do not send raw responses to clients
+- Disable HTTP redirections
+- Do not mitigate SSRF via the use of a deny list or regular expression
+
+</div>
+
+---
+
+# A10: Server Side Request Forgery: The Attack
+
+<div class="dense">
+
+- Run the server for step 5 (`cd src/a10-server-side-request-forgery`, `npm start`)
```

Comment:
```suggestion
- Run the server for step 10 (`cd src/a10-server-side-request-forgery`, `npm start`)
```

Reply:
This is slide 10 right? Or do we mean here some other kinds of steps?

Reply:
yes this is a typo